src/Core/Framework/Adapter/Twig/Filter/LeadingSpacesFilter.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Framework\Adapter\Twig\Filter;
+
+use Shopware\Core\Framework\Log\Package;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+#[Package('framework')]
```

Comment:
could you mark this class as internal, so we can it change more easily. the public API is the Twig filter name, which should not change of course, but the implementation could

Reply:
Yes that makes sense